app/uk/gov/hmrc/individualdetailsdesstub/domain/TenDigitSaUtr.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.individualdetailsdesstub.domain
+
+import uk.gov.hmrc.domain.{SimpleName, TaxIdentifier}
+
+case class TenDigitSaUtr(utr: String) extends TaxIdentifier with SimpleName {
```

Comment:
I dont think we should create our own validation but use https://github.com/hmrc/domain/blob/8c62f35561a59b2c8e900c8d2bc172a4c97e1399/src/main/scala/uk/gov/hmrc/domain/SaUtr.scala#L22 instead